docs/reference/docker-compose.md

```diff
@@ -87,15 +87,17 @@ relative to the current working directory.
 
 The `-f` flag is optional. If you don't provide this flag on the command line,
 Compose traverses the working directory and its subdirectories looking for a
-`docker-compose.yml` and a `docker-compose.override.yml` file. You must supply
-at least the `docker-compose.yml` file. If both files are present, Compose
-combines the two files into a single configuration. The configuration in the
-`docker-compose.override.yml` file is applied over and in addition to the values
-in the `docker-compose.yml` file.
+`docker-compose.yml` and a `docker-compose.override.yml` file. See also the
+`COMPOSE_FILE` [environment variable](overview.md#compose_file). You must
```

Comment:
Yeh, I struggled to make my mind up about this one, I originally had it at the end but then moved it cos it didn't feel quite right cos we're talking about multiple files. I'll move it again and see what you think.